docker/ecs/async-operation/async-operation/README.md

```diff
@@ -0,0 +1,69 @@
+# async-operation Docker image
+
+This image is intended for use with the AsyncOperation model. It will download
+an AWS Lambda function from AWS, download a payload from S3, execute the lambda
+function (inside the Docker container), then update the RDS database record for the
+AsyncOperation with the result of the lambda function.
```

Comment:
```suggestion
This image is intended for use with the AsyncOperation model. It will:
1. download an AWS Lambda function, 
2. download a payload from S3,
3. execute the lambda function (inside the Docker container), 
4. then update the RDS database record for the AsyncOperation with the result of the lambda function.
``` 

What do you think if we put it into an easy to read numbered sequence? Or we could use bullet points?

Reply:
c73caaf0cde4d2e276190d22f38a284369b1d006